sdk/digitaltwins/digitaltwins/review/digitaltwins.api.md

```diff
@@ -0,0 +1,372 @@
+## API Report File for "@azure/digitaltwins"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureServiceClientOptions } from '@azure/core-arm';
+import * as coreHttp from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+import { ServiceClientCredentials } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface AzureDigitalTwinsAPIOptions extends AzureServiceClientOptions {
+    // (undocumented)
+    baseUri?: string;
+}
+
+// @public
+export interface DigitalTwinModelsAddOptionalParams extends coreHttp.RequestOptionsBase {
+    models?: any[];
+}
+
+// @public
+export type DigitalTwinModelsAddResponse = Array<ModelData> & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: ModelData[];
+    };
+};
+
+// @public
+export type DigitalTwinModelsGetByIdResponse = ModelData & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: ModelData;
+    };
+};
+
+// @public
+export interface DigitalTwinModelsListOptionalParams extends coreHttp.RequestOptionsBase {
+    dependenciesFor?: string[];
+    digitalTwinModelsListOptions?: DigitalTwinModelsListOptions;
+    includeModelDefinition?: boolean;
+}
+
+// @public
+export interface DigitalTwinModelsListOptions {
+    maxItemCount?: number;
+}
+
+// @public
+export type DigitalTwinModelsListResponse = PagedModelDataCollection & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: PagedModelDataCollection;
+    };
+};
+
+// @public
+export interface DigitalTwinsAddHeaders {
+    eTag: string;
```

Comment:
js prefers `etag`.

Reply:
This would be a swagger change but it would effect all languages.

---
In reply to: [455321769](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r455321769) [](ancestors = 455321769)

Reply:
I don't know if swagger has an "etag" type, but in the least it could be a fix to autorest code generator to use etag type if the parameter name is etag.

Reply:
As a data point, currently in the data plane swagger, the etag header is spelled "ETag" (capital E and capital T). This looks consistent with the spelling in the RFC (RFC 7232) and places like Wikipedia.